### DISCUSSION TITLE
AttributeError: 'Series' object has no attribute 'to_list'

### DISCUSSION THREAD

**Discussion by GuobaoXu** [asreview/asreview#1293] [Q&A]
When  I run the AsreviewLab in Oracle mode, I upload the data and selected the relevant and irrelavent data, I selected the model as: Naive Bayes + TF-IDF + Maximum + Dynamic resampling, but it showed "AttributeError: 'Series' object has no attribute 'to_list' ". Does anyone know how to fix this error? Does this related to model selection or others?

**Comment by Rensvandeschoot** [comment]
Further discussed in https://github.com/asreview/asreview/issues/1294
